index.js

```diff
@@ -35,26 +35,35 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
   const account = accounts[0];
   const networkId = await web3.eth.net.getId();
 
-  // Example EMP Parameters. Customize these.
-  const empParams = {
-    expirationTimestamp: "1640995200", // 01/01/2022 @ 0:00 (UTC)
+  // Example Perpetual Parameters. Customize these.
+  const perpetualParams = {
     collateralAddress: "0xd0a1e359811322d97991e03f863a0c30c2cf029c", // Kovan WETH address. Mainnet WETH is: 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2.
     priceFeedIdentifier: padRight(utf8ToHex("USDETH"), 64), // Using the USDETH price.
-    syntheticName: "uUSDwETH Synthetic Token Expiring 1 January 2022", // Long name.
-    syntheticSymbol: "uUSDwETH-JAN", // Short name.
+    fundingRateIdentifier: padRight(utf8ToHex("USDETH"), 64), // Using the USDETH funding rate identifier.
```

Comment:
The funding rate identifier should be different from the regular identifier. We don't have any of these approved yet (and we probably won't for a bit after launch).

However, I could imagine us using the _same_ funding rate identifier for all perp contracts that want to use the same funding rate computation, since the perp contracts pass in their token address as a part of the ancillary data. Definitely more to think about here, but for now, I think it makes sense to change this to something like `USDETH_FR`, or similar along with a TODO to update it when we have an approved funding rate identifier.

Reply:
That makes sense. I'll probably revise to do something similar to what @smb2796 did in launch-emp, where these are passed as arguments.

Reply:
I like that plan